packages/client/components/ActionSidebarAgendaItemsSection.tsx

```diff
@@ -1,4 +1,4 @@
-import styled from '@emotion/styled'
+import styled from '@emotion/styled-base'
```

Comment:
why the -base?

Reply:
Huh, that must be the auto-import being strange. I've updated it!